core/src/main/resources/lib/layout/layout.jelly

```diff
@@ -66,18 +70,21 @@ ${h.initPageVariables(context)}
   this also allows us to configure HttpSessionContextIntegrationFilter not to create sessions,
   which I suspect can end up creating sessions for wrong resource types (such as static resources.)
 -->
+<j:if test="${attrs.type==null}">
+  <j:set var="layoutType" value="two-column"/>
+</j:if>
 <j:set var="isMSIE" value="${userAgent.contains('MSIE')}"/>
 <j:set var="_" value="${request.getSession()}"/>
 <j:set var="_" value="${h.configureAutoRefresh(request, response, attrs.norefresh!=null and !attrs.norefresh.equals(false))}"/>
-  <j:set var="extensionsAvailable" value="${h.extensionsAvailable}"/>
-  <j:if test="${request.servletPath=='/' || request.servletPath==''}">
-    ${h.advertiseHeaders(response)}
-    <j:if test="${extensionsAvailable}">
-      <j:forEach var="pd" items="${h.pageDecorators}">
-        <st:include it="${pd}" page="httpHeaders.jelly" optional="true"/>
-      </j:forEach>
-    </j:if>
+<j:set var="extensionsAvailable" value="${h.extensionsAvailable}"/>
+<j:if test="${request.servletPath=='/' || request.servletPath==''}">
+  ${h.advertiseHeaders(response)}
+  <j:if test="${extensionsAvailable}">
+    <j:forEach var="pd" items="${h.pageDecorators}">
+      <st:include it="${pd}" page="httpHeaders.jelly" optional="true"/>
+    </j:forEach>
```

Comment:
@tfennelly Right, I think it provides more readability in this block where we are setting variables.